app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Project.scala

```diff
@@ -27,6 +32,9 @@ case class Project(
 /** Case class for project creation */
 object Project extends GeoJsonSupport {
 
+  /* One week, in milliseconds */
+  val DEFAULT_CADENCE: Long = 604800000
```

Comment:
`aoiCadence` doesn't live in an `Option`, so the front-end would always have to set it. In which case yeah, the value wouldn't be needed here.